packages/space-opera/src/components/model_viewer_preview/types.ts

```diff
@@ -41,3 +42,13 @@ export interface GltfState {
   gltf?: GltfModel;
   gltfJsonString: string;
 }
+
+export const INITIAL_MODEL_STATE: ModelState = {
+  thumbnailsById: new Map<string, string>(),
```

Comment:
I notice the previous texturesById was a Map<string, Texture> , why is this one a Map<string, string>?

Reply:
Nvm, read texturesById as thumbnailsById